tests/test_user_panel.py

```diff
@@ -23,7 +24,7 @@
 )
 
 from tests.common_tools import make_user
-
+from .test_cart import _setup
```

Comment:
Not a big fan of this one, maybe it's time to export this _setup to some common module, and call it `def setup_conference_with_typical_fares` or copy the code here and also call it _setup? I'm thinking about avoiding weird, non obvious dependencies, like private functions from another module. WDYT?

Reply:
I like the idea of moving it to a common module and renaming it; it makes it clearer what it does :)